setup.py

```diff
@@ -13,11 +13,17 @@
         'agasc-magnitudes-report=agasc.scripts.mag_estimate_report:main',
         'agasc-update-magnitudes=agasc.scripts.update_mag_supplement:main',
         'agasc-update-supplement=agasc.scripts.update_supplement:main',
+        'agasc-supplement-tasks=agasc.scripts.supplement_tasks:main',
     ]
 }
 
 
-data_files = [(os.path.join('share', 'agasc'), ['task_schedule.cfg'])]
+data_files = [(
+    os.path.join('share', 'agasc'),
+    ['task_schedules/task_schedule_supplement_dispositions.cfg',
+     'task_schedules/task_schedule_promote_supplement.cfg',
```

Comment:
setup can't find this one

Reply:
ah! yes. I removed the promote task in the last commit and forgot to remove this one. Just fixed it.

Reply:
thanks!  That's what I figured from the workflow.